### DISCUSSION TITLE
wont let me add a skyblock world

### DISCUSSION THREAD

**Discussion by threlief** [GriefPrevention/GriefPrevention#846] [Q&A] [answered]
i add my bskyblock to the config worlds area for claim to work and it keeps erasing it i even used the config from my other skyblocks server and still not working i cant figure out y?

**Comment by RoboMWM** [comment]
is it loaded after GP?

**Comment by threlief** [comment]
yes

**Comment by RoboMWM** [comment]
K that's the problem then, GP is designed to deal with worlds when it enables, so if the developer could add a `loadbefore` to their `plugin.yml` for GP that'd help.

Else, I guess we could add yet another `softdepend`...

**Comment by threlief** [comment]
ok we uninstalled and reinstalled and it still wont lt us add it keeps clearing it on restart

**Comment by bigpresh** [comment]
I think what @RoboMWM meant wasn't "had you installed it before GP", but "is the plugin loading before GP when your server starts up".

If the latter, when GP parses its config file, it's going to see a world that doesn't exist as far as it's concerned (because it hasn't yet loaded), so it can't protect it.  I assume that on your other skyblocks server you're using a different way to manage the skyblock world, and that does load before GP?

The best solution, as RoboMWM said, is for the bskyblock plugin to add a `loadbefore` to their plugin.yml file which states that it should be loaded before GriefPrevention (and any other protection-type plugins, probably).

If they're not willing to do so, we could instead add it to the already quite long list of soft dependencies in `softdepend` in [GriefPrevention's plugin.yaml](https://github.com/TechFortress/GriefPrevention/blob/master/src/main/resources/plugin.yml#L3)

**Comment by Jikoo** [comment, marked as answer]
GP should be added to BentoBox's loadbefore - if this happens with BSkyBlock, it will happen with any other BentoBox addon.

Please open an issue on BentoBox's tracker requesting compatibility.
